### PR TITLE
AADS, on the CLI, when I launch kili project describe, I have an easy-to-read project description

### PR DESCRIPTION
**Pull request by theodu**
- [ ] I opened a merge request on `kili` on a branch with the same name than the branch of the current pull request and forced security tests.
- [ ] I opened a pull request on the Node playground if the changes are breaking.

### CODE REVIEW

File: kili/cli.py [hunk 1]
```diff
@@ -102,6 +104,45 @@ def import_assets(api_key: str,
...
+                                 fields=['title', 'id', 'description', 'numberOfAssets',
+                                         'numberOfRemainingAssets', 'numberOfReviewedAssets',
+                                         'numberOfAssetsWithSkippedLabels',
+                                         'honeypotMark', 'consensusMark',
+                                         'numberOfOpenIssues', 'numberOfSolvedIssues',
+                                         'numberOfOpenQuestions', 'numberOfSolvedQuestions'])
+    except:
+        # pylint: disable=raise-missing-from
+        raise NotFound(f'project ID: {project_id}')
+    infos, dataset_statistics, quality_statistics = get_project_statistics(
```
Comment (PierreLeveau) [inline kili/cli.py:136]: `get_project_metrics` ?

File: kili/queries/project/helpers.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+"""
+Helpers for the project queries.
+"""
+
+from typing import List, Tuple
+
+
+def get_project_statistics(project: dict) -> Tuple[List[list], List[list], List[list]]:
+    """Get statisctics for a project.
```
Comment (PierreLeveau) [inline kili/queries/project/helpers.py:9]: typo

**Review by PierreLeveau — CHANGES_REQUESTED** [review]
I suggest to change statistics to metrics (more widely adopted)

File: kili/cli.py (hunk 1 above)
Comment (theodu) [reply kili/cli.py:136]: Yes, i'll change that

File: kili/cli.py [hunk 3]
```diff
@@ -150,6 +151,45 @@ def import_assets(api_key: str,
...
+              help='Your Api Key')
+@click.option('--endpoint', type=str,
+              default='https://cloud.kili-technology.com/api/label/v2/graphql',
+              help='The API Endpoint')
+@click.option('--project-id', type=str, required=True,
+              help='Id of the project to import assets in')
+def describe_project(api_key: str,
+                     endpoint: str,
+                     project_id: str):
+    """Show Analytics of a project.
```
Comment (crsegerie) [inline kili/cli.py:165]: cool command!

File: kili/cli.py [hunk 4]
```diff
@@ -150,6 +151,45 @@ def import_assets(api_key: str,
...
 
 
+@project.command(name="describe")
+@click.option('--api-key', type=str, envvar='KILI_API_KEY', required=True,
+              help='Your Api Key')
+@click.option('--endpoint', type=str,
+              default='https://cloud.kili-technology.com/api/label/v2/graphql',
+              help='The API Endpoint')
+@click.option('--project-id', type=str, required=True,
+              help='Id of the project to import assets in')
```
[the remark this answers is not in the document]
Comment (theodu) [reply kili/cli.py:161]: That is an excellent idea! I suggest to take it in the next command as there is still one command left to implement.

**Review by crsegerie — APPROVED** [review]

**Review by PierreLeveau — APPROVED** [review]

**Pull request merged**